packages/traceability-tests/tests/privateKeyJwk.js

```diff
@@ -0,0 +1,8 @@
+const privateKeyJwk = {
+  kty: 'OKP',
+  crv: 'Ed25519',
+  x: 'zX2zoGmW36TTL_kw3g-KFVjh5IoaDcoSc08jID4uGrs',
+  d: '95hNcacuqtWQP70MDYrL6DRUJ8iGg-oq13I0KM5Etw8',
+};
+
+module.exports = privateKeyJwk;
```

Comment:
This works, but perhaps p256?

Reply:
I can change that part, it just makes the examples longer.